src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/WebWorkerTest.cs

```diff
@@ -376,6 +377,7 @@ public async Task ThreadingTimer(Executor executor)
                 await tcs.Task;
             }, cts.Token);
 
+            Console.WriteLine("ThreadingTimer: ManagedThreadId: " + Environment.CurrentManagedThreadId + " NativeThreadId: " + WebWorkerTestHelper.NativeThreadId);
```

Comment:
`WebWorkerTest.ThreadingTimer` didn't hit at all in the [log](https://github.com/dotnet/runtime/issues/98216). So it would be also good to print timestamp before (line 367) and after (380)

Maybe we just need to move `hit = true;` before `tcs.SetResult();` on line 374, because that's race conditon in the test.